theforeman.org/scripts/debian/stage_packages.sh

```diff
@@ -7,19 +7,46 @@ set -xe
 # * the web node must have the freight class applied
 
 # Find the deps in the build-dir from the previous step
-DEB_PATH=./debian/${os}/build-${project}
+for possible_path in ./debian/${os}/build-${project} ./plugins/build-${project} ./dependencies/${os}/build-${project}; do
+  if [[ -d "${possible_path}" ]]; then
+    DEB_PATH="${possible_path}"
+  fi
+done
 
 # Upload all builds to stagingdeb for testing
-echo "scratch build: uploading to stagingdeb/${os}/${repoowner}-${version}"
+HOSTS="web01.osuosl.theforeman.org"
 export RSYNC_RSH="ssh -i /home/jenkins/workspace/staging_key/rsync_freightstage_key"
 USER=freightstage
-HOSTS="web01.osuosl"
-COMPONENT=${repoowner}-${version}
+SUITE=${os}
```

Comment:
@ekohl I am not sure if this is what you intended by using `SUITE`

Reply:
Yes, this is exactly what I meant.